.evergreen/.evg.yml

```diff
@@ -599,16 +599,6 @@ functions:
           LOGIN_CONTEXT_NAME=${LOGIN_CONTEXT_NAME} \
           .evergreen/run-gssapi-auth-test.sh
 
-  "run mmapv1 storage test":
```

Comment:
4.0 is the last server version that supports this storage engine, so removing the task.